debian/libibumad3.install

```diff
@@ -1 +1,2 @@
 usr/lib/*/libibumad*.so.*
+lib/udev/rules.d/libibumad.rules
```

Comment:
Yes. It's fine. For multi-arch, you just have to ensure that all these files that share the same path have the same content (which is true in this case).